requirements.txt

```diff
@@ -1,4 +1,6 @@
+argon2-cffi==20.1.0
```

Comment:
Bist du sicher, dass das mit rein muss? Ich hab bei mir python-magic installiert und beim `freeze` wurde mir das nicht in der `requirements.txt` angezeigt...

Reply:
Keine Ahnung, wie die ganzen Dependencies da rein gekommen sind... Ich versuch die mal zu deinstallieren.